src/MakePotCommand.php

```diff
@@ -41,7 +41,7 @@ class MakePotCommand extends WP_CLI_Command {
 	/**
 	 * @var array
 	 */
-	protected $exclude = [ 'node_modules', '.git', '.svn', '.CVS', '.hg', 'vendor', 'Gruntfile.js', 'webpack.config.js', '*.min.js' ];
+	protected $exclude = [ 'node_modules', '.git', '.svn', '.CVS', '.hg', 'vendor', 'Gruntfile.js', 'webpack.config.js' ];
```

Comment:
As #87 was now merged, this change is not needed anymore.

By default, minified files should still be excluded, and if you want to explicitly include them, you can do so by adding `--include=*.min.js` to the command.

Reply:
Awesome! Will remove this in that case.